graphql_compiler/compiler/expressions.py

```diff
@@ -227,6 +227,19 @@ def to_cypher(self):
         # Cypher has built-in support for variable expansion, so we'll just emit a variable
         # definition and rely on Cypher to insert the value.
         self.validate()
+
+        # The Neo4j client allows us to pass dates and datetimes directly as arguments. See the
```

Comment:
Looks good to me. Just one nit: date objects and datetime objects, (dates and datetimes are not types in python and we want to be as clear as possible that we are referring to actual python types).

Reply:
done, ready to go